### ISSUE TITLE
The `<code>` tag, when inline code with backticks, does not receive a background

### ISSUE THREAD

**Issue by rockcavera** [pietroppeter/nimibook#89]
All inline code with backticks is not being rendered with a background, which makes it difficult to distinguish an inline code from the rest of the text.

This problem occurs in all nimibook themes.

**Comment by pietroppeter** [comment]
Yes, it is a problem of nimib. Nim code in nbCode or similar blocks is statically highlighted and receive the different background from highlight.js default theme. The code in markdown is not statically highlighted and highlight.js theme is not applied. I think there should be an issue in nimib related to this (currently on mobile and hard to find it).

**Comment by rockcavera** [comment]
From the research I did, adding `class="hljs"`, in the `<code>` tags, of the inline code, would solve the problem.

However, in a quick look at the `nimibook` code, I saw that the rendering is all delegated to `nimib`. I haven't seen the `nimib` code yet, but I believe that the change cannot be made there, as it will disrupt the `nimib` library.

**Comment by pietroppeter** [comment]
Yes adding that class would trigger the highlighting through css. The change should be indeed made in nimib. Nimibook is not responsible for collecting multiple nimib documents in a single static site with a predefined theme (from mdbook)
